BACKEND/userAuth/src/main/java/com/apptime/auth/service/TaskManagerService.java

```diff
@@ -3,13 +3,13 @@
 import com.apptime.auth.config.TaskStateMachine;
 import com.apptime.auth.model.Task;
 import com.apptime.auth.model.TaskState;
+import com.apptime.auth.repository.TaskReportRepository;
```

Comment:
I would keep the two service separate to make testing easy. They are parallel. they can both depend on lower level layer. Data access layer.